### PR TITLE
Add nsp

### PR DESCRIPTION
**Pull request by ellerbrock**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -582,7 +582,7 @@
 ### Security
 
 - [snyk](https://github.com/Snyk/snyk) - CLI and build-time tool to find & fix vulnerable npm dependencies.
-
+- [nsp](https://nodesecurity.io/opensource) - CLI to the Node Security Platform.
```
Comment (sindresorhus) [inline readme.md]: Can you link to the github repo instead?

Comment (sindresorhus) [inline readme.md]: The description could be clearer about what the CLI actually does.

File: readme.md [hunk 2]
```diff
@@ -582,7 +582,7 @@
 ### Security
 
 - [snyk](https://github.com/Snyk/snyk) - CLI and build-time tool to find & fix vulnerable npm dependencies.
-
```
Comment (sindresorhus) [inline readme.md]: Don't remove the newline.

Comment (ellerbrock) [inline readme.md]: ok i just added the line and made a new pr. hope its ok like this ...

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome-nodejs/pull/523#discussion_r65940926

**Comment by ellerbrock** [comment]
changed the link to the github repo and updated the description

File: readme.md [hunk 3]
```diff
@@ -582,6 +582,7 @@
 ### Security
 
 - [snyk](https://github.com/Snyk/snyk) - CLI and build-time tool to find & fix vulnerable npm dependencies.
+- [nsp](https://github.com/nodesecurity/nsp) -  CLI tool to help and identify known vulnerabilities in your projects.
```
Comment (SamVerschueren) [inline readme.md]: 2 spaces between `-  CLI`

**Comment by ellerbrock** [comment]
ok second space gone :)

**Pull request closed without merging**